mobile-academy/src/main/java/org/motechproject/nms/mobileacademy/service/impl/CourseNotificationServiceImpl.java

```diff
@@ -40,7 +39,7 @@
     private static final String DELIVERY_STATUS = "deliveryStatus";
     private static final String ADDRESS = "address";
     private static final String SMS_CONTENT_PREFIX = "sms.content.";
-    private static final String SMS_DEFAULT_LANGUAGE_CODE = "50";
+    private static final String SMS_DEFAULT_LANGUAGE_CODE = "default";
```

Comment:
that's the name of the property, not the value, right?
wanna rename it to show that?